CheatSheetVsCodePowerShell.md

```diff
@@ -0,0 +1,63 @@
+# VSCode Cheat Sheet – PowerShell
+## Specific PowerShell Commands
+
+|Command | Explanation |
+| --- | --- |
+|**CTRL + Shift + P (F1) PowerShell**|![alt text](./images/commandPallette.png "Use this to find commands specific for PowerShell")   | 
+| **SHIFT + ALT + E** | Expand current alias |
+| **F8** | Run selection |
+| **CTRL + F1** | Get Online Help for Command |
+| **CTRL + ALT + P** | Start PowerShell |
+| **CTRL + ALT + U** | Start PowerShell Unrestricted |
+| **CTRL + shift + i** | Open Current file in PowerShell ISE |
+| **CTRL + Alt + O** | Open new Terminal |
+| **CTRL + SHIFT + `** | Create new integrated terminal |
+| **CTRL + `** | Change to active integrated terminal |
+| **&lt;# + Tab** | Automatically generate Help comments |
+| **Snippets** | In editor use Name + tab |
+## General Commands 
```

Comment:
Rather than list all of these for VSCode (Windows only), we may way to vector folks to the VSCode official keyboard shortcut references for Windows, Linux and macOS: https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-shortcuts-reference